app/src/main/java/fr/free/nrw/commons/contributions/Contribution.java

```diff
@@ -220,9 +220,10 @@ private String licenseTemplateFor(String license) {
                 return "{{self|cc-by-3.0}}";
             case Prefs.Licenses.CC_BY_SA:
                 return "{{self|cc-by-sa-3.0}}";
-        }
+                default:
+                    throw new RuntimeException("Unrecognized license value: " + license);
 
-        throw new RuntimeException("Unrecognized license value: " + license);
```

Comment:
I feel it's easier just like this. However if you are going to put it into switch can you decrease indentation of lines 223 and 224 to match the rest.